datahub-web-react/src/app/entity/domain/DataProductsTab/DataProductAdvanceOption.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Collapse, Form, Input, Typography } from "antd";
+import styled from "styled-components";
+import { validateCustomUrnId } from '../../../shared/textUtil';
+import { DataProductBuilderFormProps } from "./types";
+
+
+const FormItem = styled(Form.Item)`
+    .ant-form-item-label {
+        padding-bottom: 2px;
+    }
+`;
+
+const FormItemWithMargin = styled(FormItem)`
+    margin-bottom: 16px;
+`;
+
+const FormItemNoMargin = styled(FormItem)`
+    margin-bottom: 0;
+`;
+
+const AdvancedLabel = styled(Typography.Text)`
+    color: #373d44;
+`;
+
+export function DataProductAdvanceOption({builderState, updateBuilderState }:DataProductBuilderFormProps){
```

Comment:
naming nitpick: 

Let's call this DataProductAdvancedOption

Reply:
Done